indico/modules/events/management/controllers/protection.py

```diff
@@ -63,42 +70,86 @@ def _process(self):
                                 endpoint='event_management.acl')
 
 
-class RHEventProtection(RHManageEventBase):
+class RHEventProtectionBase(RHManageEventBase):
+    def __init__(self):
```

Comment:
I would just just write a `get_permissions_info()` function that returns a `whitelisted_permissions, permissions_tree` tuple (btw, isn't `available_permissions` a better name than `whitelisted_permissions`?).

This would also make it easier in the future if we want to use the same widget e.g. for category permissions.